package.json

```diff
@@ -22,14 +25,23 @@
     "nodemon": "^1.17.3",
     "pa11y": "^5.0.3",
     "puppeteer": "^1.2.0",
+    "rollup": "^0.58.2",
+    "rollup-plugin-commonjs": "^9.1.2",
+    "rollup-plugin-json": "^2.3.0",
+    "rollup-plugin-node-resolve": "^3.3.0",
     "serve-static": "^1.13.2",
+    "shx": "^0.2.2",
     "tap": "^11.1.3"
   },
   "scripts": {
     "test": "node scripts/serve",
     "watch-test": "nodemon --watch scripts --watch test -e js,html scripts/serve.js",
-    "lint": "eslint test scripts",
-    "serve": "http-server test"
+    "serve": "http-server test",
+    "lint": "eslint test scripts && eslint -c .eslintrc-amaps.js src",
```

Comment:
I think you could move `.eslintrc-amaps.js` to `src/.eslintrc.js` and then change this line into: `"lint": "eslint test scripts src",`.

Reply:
Good one! done.